### PR TITLE
Document the environment variable pass-through prefix option introduced by #382

### PR DESCRIPTION
**Pull request by reivilibre**
We seem to have forgotten to document this option, so this PR tries to put that right.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,6 +98,9 @@ If you're looking to run against a custom Dockerfile, it must meet the following
 - The homeserver needs to assume dockerfile `CMD` or `ENTRYPOINT` instructions will be run multiple times.
 - The homeserver needs to use `complement` as the registration shared secret for `/_synapse/admin/v1/register`, if supported. If this endpoint 404s then these tests are skipped.
 
+If the image needs environment variables to function, you can:
+1. define a pass-through prefix with e.g. `COMPLEMENT_SHARE_ENV_PREFIX=PASS_`; then
+2. prefix the desired environment variables with that prefix; e.g. `PASS_SYNAPSE_COMPLEMENT_USE_WORKERS=true`.
```
[the remark this answers is not in the document]
Comment (richvdh) [reply README.md:103]: > the image _shouldn't_ **need** environment variables to function.

heh. https://github.com/matrix-org/synapse/pull/13075 fixes this for the synapse image.

[the remark this answers is not in the document]
Comment (kegsay) [reply README.md:103]: Agreed, please move to the "Running" section. It should be an optional thing.

[the remark this answers is not in the document]
Comment (reivilibre) [reply README.md]: Moved!

**Review by richvdh — APPROVED** [review]
lgtm

**Pull request merged**